Screens/History.js

```diff
@@ -85,23 +74,24 @@ export default class HistoryDataBase extends React.Component {
 
     //attributes line up with the quesiton marks in the corresponding order
     hdb.transaction((tx) => {
-      tx.executeSql(
-        "insert into historyitems (name, expirationDate, price) values (?, ?, ?)",
-        [name, expirationDate, price]
-      );
-      tx.executeSql("select * from historyitems", [], (_, { rows }) =>
-        console.log(JSON.stringify(rows))
-      );
+      tx.executeSql("insert into historyitems (name, expirationDate, price) values (?, ?, ?)", [name, expirationDate, price]);
+      tx.executeSql("select * from historyitems", [], (_, { rows }) => console.log(JSON.stringify(rows)));
     }, null);
   }
 
   render() {
     return (
       <View style={styles.container}>
+        <TouchableOpacity
+          style={styles.sortBtn}

```

Comment:
Similarly here on line 86. topic is deletion and not sorting so renaming sortBtn to something more appropriate would be better.